File: AlbumLibrary/src/main/java/com/album/presenter/PreviewPresenter.java
package com.album.presenter;


import com.album.entity.AlbumEntity;

import java.util.ArrayList;
import java.util.List;

/**
 * by y on 17/08/2017.
 */

public interface PreviewPresenter {
    void scan(String bucketId, int page, int count);

    void mergeEntity(List<AlbumEntity> albumEntityList, ArrayList<AlbumEntity> selectAlbumEntityList);

}

File: AlbumLibrary/src/main/java/com/album/presenter/PreviewPresenter.kt
package com.album.presenter


import com.album.entity.AlbumEntity

import java.util.ArrayList

/**
 * by y on 17/08/2017.
 */

interface PreviewPresenter {
    fun scan(bucketId: String, page: Int, count: Int)

    fun mergeSelectEntity(albumEntityList: List<AlbumEntity>, selectAlbumEntityList: ArrayList<AlbumEntity>)
}
